simulator/src/main/java/com/hazelcast/simulator/protocol/processors/AbstractOperationProcessor.java

```diff
@@ -68,7 +61,7 @@ public final ResponseType process(SimulatorOperation operation, SimulatorAddress
                     return processOperation(operationType, operation, sourceAddress);
             }
         } catch (Throwable e) {
-            exceptionLogger.log(e);
+            LOGGER.warn(e);
```

Comment:
You should use the `ExceptionReporter` here, otherwise we might lose exception from the `WorkerOperationProcessor`.